apps/dynatrace/dynatrace-operator.yaml

```diff
@@ -27,6 +27,8 @@ spec:
           key: kubernetes.io/os
           operator: Equal
           value: "windows"
+      nodeSelector:
+        agentpool: linux
```

Comment:
does it work on ARM?

Reply:
Haven't tested it - there does seem to be arm based images for it. Only got arm based nodes in sbox atm anyway

Reply:
Just noticed, there is one csi driver daemonset scheduled on the system node as well so I need to add that too

Reply:
Changed it to target linux os nodes which is the inverse way of achieving what the existing tolerations already should be doing. Still not sure why it's trying to schedule when it's not supposed to